src/settings/SettingsModal.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
This is for `e.target.checked`. EventTarget does not have `checked` defined. So `// @ts-ignore` is used.